sparse/ndsparse/tensor.py

```diff
@@ -0,0 +1,9 @@
+import pytaco as pt
+from pytaco import compressed
+import numpy as np
+from numpy.lib.mixins import NDArrayOperatorsMixin
+
+
+class tensor(np.ndarray, pt.tensor, NDArrayOperatorsMixin):
```

Comment:
We should wrap `pt.tensor`, not inherit from it. And `np.ndarray` shouldn't be involved at all.